src/test/java/LottoTest.java

```diff
@@ -1,35 +1,49 @@
+import domain.LottoTickets;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
 
-import domain.*;
+import domain.Lotto;
+import domain.LottoNumber;
+import domain.LottoService;
 
 
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-//실패코드 작성
 public class LottoTest {
     @Test
-    @DisplayName("로또 숫자 정렬 아닐때")
-    void testLottoSort() {
-        List<Integer> unsorted = Arrays.asList(5, 1, 3, 2, 4, 6);
-        assertThrows(IllegalArgumentException.class, () -> new Lotto(unsorted));
-
+    @DisplayName("로또 숫자가 1~45 사이의 숫자가 아닐 경우 예외가 발생한다")
+    void testLottoRange() {
+        IllegalArgumentException e = assertThrows(IllegalArgumentException.class, () -> {
+            List<LottoNumber> outOfRange = Arrays.asList(new LottoNumber(0), new LottoNumber(2), new LottoNumber(3), new LottoNumber(4), new LottoNumber(5), new LottoNumber(6));
+            new Lotto((SortedSet<LottoNumber>) outOfRange);
+        });
+        assertEquals("로또 번호는 1부터 45 사이의 숫자여야 합니다.", e.getMessage());
     }
 
     @Test
-    @DisplayName("로또 숫자가 1~45 사이의 숫자 아닐때")
-    void testLottoRange() {
-        List<Integer> outOfRange = Arrays.asList(0, 2, 3, 4, 5, 6);
-        assertThrows(IllegalArgumentException.class, () -> new Lotto(outOfRange));
+    @DisplayName("보너스볼 숫자가 기존 당첨번호와 중복될 경우 예외가 발생한다.")
+    void bonusBallDuplicate() {
+        LottoService lottoService = new LottoService();
+        Lotto lottoAnswer = lottoService.parseLottoAnswer("1,2,3,4,5,6");
+        IllegalArgumentException e = assertThrows(IllegalArgumentException.class, () -> {
+            lottoService.validateBonusBall(lottoAnswer, new LottoNumber(1));
+        });
+        assertEquals("보너스 볼은 당첨 번호와 중복될 수 없습니다.", e.getMessage());
     }
 
     @Test
-    @DisplayName("로또 숫자가 중복되는지")
-    void testLottoDuplicate() {
-        List<Integer> duplicated = Arrays.asList(1, 2, 3, 3, 4, 5);
-        assertThrows(IllegalArgumentException.class, () -> new Lotto(duplicated));
+    @DisplayName("로또 번호 수동 입력 시 중복된 숫자가 들어올 경우 예외가 발생한다.")
+    void manualLottoDuplicate() {
+        SortedSet<LottoNumber> duplicateNumbers = new TreeSet<>(Arrays.asList(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3), new LottoNumber(4), new LottoNumber(5), new LottoNumber(5)));
```

Comment:
매우 많은 수의 인자를 받는 경우 한 줄에 모든 인자를 적기보다 아래와 같이 줄바꿈을 통해 파악하기 쉬운 형태로 작성해보면 어떨까요?
아래와 같은 포맷팅은 많은 인자를 받을 때 뿐만 아니라 `Stream`을 활용한 메서드 체이닝을 활용할 때도 한 줄에 모두 작성했을 때와 줄바꿈을 통해 여러 줄에 걸쳐 작성을 했을 때 가독성이 더 좋은 코드를 작성할 수 있어요! (참고 하셔서 다른 줄도 같이 수정 진행해주시면 좋겠네요!)

```suggestion
        SortedSet<LottoNumber> duplicateNumbers = new TreeSet<>(
            Arrays.asList(
                new LottoNumber(1), 
                new LottoNumber(2), 
                new LottoNumber(3), 
                new LottoNumber(4), 
                new LottoNumber(5), 
                new LottoNumber(5)
            )
        );
```

Reply:
네 조언 감사합니다!! 수정하겠습니당😆

Reply:
이 부분 인텔리제이에서 자동 정렬 하는 게 있더라구요..!
always는 너무 사소한 단위까지 잘라 조금 번잡해 보이는 것 같아서 if long 일때로 해  두었습니다
<img width="200" height="144" alt="image" src="https://github.com/user-attachments/assets/a75147e8-af35-4699-9fef-08fa50fe97fa" />

Reply:
지금은 인텔리제이의 자동정렬로 충분할 수 있지만, 협업 혹은 직무에 나가서는 특정 코드 컨벤션을 정하고 해당 컨벤션에 맞게 작업을 하실 수도 있어요! 

만약, 시간이 되신다면 코드 컨벤션에 관한 내용을 함께 찾아보면 많은 도움이 되실 것 같습니다 😄